src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+    }
+    }
+    public static String findTaskDateTime(String readInput){
+        String afterSlash = readInput.substring(readInput.indexOf("/"));
+        String[] splitString = afterSlash.split("\\s+");
+        return afterSlash.substring(splitString[0].length()+1);
+    }
+
+    public static void taskCountMessage(ArrayList<Task> taskList){
+        System.out.println("Now you have " + taskList.size() + " tasks in the list."
+                + "\n"
+                + "____________________________________________________________\n");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        ArrayList<Task> taskList = new ArrayList<>();
+        taskList.add(new Todo("read book"));
+        taskList.add(new Deadline("return book", "June 6th"));
+        taskList.add(new Event("project meeting", "Aug 6th 2-4pm"));
+        taskList.add(new Todo("join sports club"));
+        taskList.get(0).setMarked();
+
+        String border = "____________________________________________________________\n";
+
+        String helloMessage = border
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n"
+                + "\n"
+                + border;
+        System.out.println(helloMessage);
+
+        boolean isExit = false;
+        Scanner userInput = new Scanner(System.in);
+        while (!isExit) {
+            String readInput;
+            while ((readInput = userInput.nextLine()) != null) {
+                String command = readCommand(readInput);
+                switch(command){
+                case "bye":
+                    System.out.println(border
+                            + "Bye. Hope to see you again soon!\n"
+                            + border);
+                    isExit = true;
+                    break;
+                case "list":
+                    if (!taskList.isEmpty()) {
+                        System.out.println(border);
+                        for (int i = 0; i < taskList.size(); i++) {
+                            System.out.println((i + 1) + ". " + taskList.get(i).toString());
+                        }
+                        System.out.println(border);
+                    } else{
+                        System.out.println(border
+                                + "You have no tasks.\n"
+                                + border);
+                    }
+                    break;
+                case "mark":
+                    getTaskFromNumber(readInput, taskList).setMarked();
+                    break;
+                case "unmark":
+                    getTaskFromNumber(readInput, taskList).setUnmarked();
```

Comment:
SLAP is off here.
Additionally, there are quite a few levels of nesting. 
You can extract out certain things as methods and improve the SLAP